### PR TITLE
SA1005 Single line comment should begin with a space

### PR DESCRIPTION
**Pull request by elachlan**
Relates to #7174
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1005.md

Used automated code fix. Sorry for the large amount of changes.

### CODE REVIEW

File: src/Shared/BuildEnvironmentHelper.cs [hunk 1]
```diff
@@ -459,8 +459,8 @@ internal static void ResetInstance_ForUnitTestsOnly(BuildEnvironment buildEnviro
 
         private static class BuildEnvironmentHelperSingleton
         {
-            // Explicit static constructor to tell C# compiler
```
Comment (Forgind) [inline src/Shared/BuildEnvironmentHelper.cs:462]: What is this? It looks like you deleted a space and replaced it with a space?

**Review by Forgind — APPROVED** [review]

File: src/Shared/BuildEnvironmentHelper.cs (hunk 1 above)
Comment (elachlan) [reply src/Shared/BuildEnvironmentHelper.cs:462]: It was apparently a "NO-BREAK SPACE", which was replaced with a "SPACE".

**Review by Nirmal4G — APPROVED** [review]
Awesome!

**Review by rainersigwald — APPROVED** [review]
This is so much churn for so little benefit. I would rather not, personally. But sure.

**Comment by elachlan** [comment]
> This is so much churn for so little benefit. I would rather not, personally. But sure.

Thanks @rainersigwald, Sorry for the increased churn and extra work load.

**Pull request merged**